dist/js/slider/slider.js

```diff
@@ -0,0 +1,31 @@
+export default class Slider {
+    constructor(element) {
+        this.firstSliderControlsNode = element.children[0];
+        this.firstSliderContentNode = element;
+        this.firstSliderControls = Array.from(this.firstSliderControlsNode.children);
+        this.firstSliderContent = Array.from(this.firstSliderContentNode.children);
+    }
+
+    addSliderEventListeners() {
+        this.firstSliderControlsNode.addEventListener('click', (event) => {
+            if (!event.target.classList.contains('controls__control')) {
+                return;
+            }
```

Comment:
What will happen if an element with class `.controls__control` will have `span` inside and user clicks on this `span`?

Reply:
fixed